### PR TITLE
#372 - wrapping tags, align padding/height/border radius with magnetic

### LINKED ISSUE #372: Update `ATag` component to handle long tags
**Describe the bug**

We have reported issue https://github.com/advthreat/intelligence/issues/176 in the Intelligence app. The reported issue complains about a long tag being wrongly displayed, with the text extending outside of the tag box. However, this application only uses the `ATag` component and inserts long text into it. Therefore, the issue should be correctly fixed in magna-react.

**To Reproduce**

1. Take example file `ATag.mdx`
2. Add new or modify current tag ("User tag") - add very long text into it, e.g.:
```
      <ATag className="cisco-blue white--text ma-1">
        <AIcon left>user</AIcon>User Tag VERY VERY LONG TAG WHICH CONTAINS BREAKABLE SPACES SO IT CAN BE DIVIDED TO MULTIPLE LINES
        <AIcon
          right
          className="cursor-pointer"
          onClick={() => alert("Close icon clicked!")}>
          close
        </AIcon>
      </ATag>
```
3. Check `ATag` example page.

**Current behavior**

Tag text is rendered outside of the tag box (see screenshot). 

**Expected behavior**

Tag is rendered well according to "Tag wrapping" section in Magnetic design:  https://magnetic.cisco.com/0a43ab5cd/p/716124-tag/t/625d3c

**Screenshots**

Screenshot 2023-05-10 at 13 18 53

**Environment (please complete the following information):**
 - Device: Desktop
 - OS: macOS
 - Browser Chrome
 - Version 112.0.5615.49

**Additional context**

Also from "Tag wrapping" is not entirely clear how should be tags displayed - when I suppose tags are inline elements we can either let tag to be filled horizontally, e.g. it can look like this:

Screenshot 2023-05-10 at 13 32 14

or e.g. vertically with maximum width:
image

We should consult with UX how this should be updated.

### PR DESCRIPTION
**Pull request by rwharrisjr**
Resolves #372 

Screenshot 2023-05-17 at 10 39 42 AM

### CODE REVIEW

**Comment by rwharrisjr** [comment]
I think this adds in wrapping while maintaining alignment of icons. Also adjusts for the magnetic design size/padding changes.

File: framework/components/ATag/ATag.mdx [hunk 1]
```diff
@@ -90,6 +90,22 @@ import {ATag} from "@cisco-sbg-ui/magna-react";
 `}
 />
 
+## Wrapping Extra Long Tags
+
+<Playground
+  code={`() => {
+  const [selectedOption, setSelectedOption] = useState("Option 1");
+  const [active, setActive] = useState(false);
+  const tagRef = useRef(null);
```
Comment (mvelkCisco) [inline framework/components/ATag/ATag.mdx:99]: Unnecessary. Probably copy & paste.

File: framework/components/ATag/ATag.mdx [hunk 2]
```diff
@@ -90,6 +90,22 @@ import {ATag} from "@cisco-sbg-ui/magna-react";
...
+  return (
+    <div style={{minHeight: 120}}>
+      <ATag level="information">This is a really really really really really really really really really really really really really really really really really really really really really really really really long tag</ATag>
```
Comment (mvelkCisco) [inline framework/components/ATag/ATag.mdx:102]: Lol, you stole my really really really long text example 😀

File: framework/components/ATag/ATag.mdx (hunk 1 above)
Comment (rwharrisjr) [reply framework/components/ATag/ATag.mdx]: Yup. Removed.

**Review by mvelkCisco — APPROVED** [review]

**Pull request merged**